<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolleyKindomForm</class>
 <widget class="QWidget" name="VolleyKindomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>VolleyKindomForm</string>
  </property>
  <widget class="QPushButton" name="Buttom_Im_player">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>報名臨打</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Buttom_Im_staff">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>工作人員</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
